REST/src/main/java/pl/jakubowskiprzemyslaw/REST/controllers/PlayerConfigController.java

```diff
@@ -0,0 +1,40 @@
+package pl.jakubowskiprzemyslaw.REST.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import pl.jakubowskiprzemyslaw.REST.models.Game;
+import pl.jakubowskiprzemyslaw.REST.models.Player;
+import pl.jakubowskiprzemyslaw.REST.services.QueueService;
+import pl.jakubowskiprzemyslaw.REST.services.SessionService;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class PlayerConfigController extends RESTController {
+
+  @Value("${queue.playerConfig}")
```

Comment:
Those properties are in fact queueName.playerConfig and so on, not queues itself. We should think about better names for those

Reply:
changed to queueName.playerConfig and other properties for queues look the same